ros2topic/ros2topic/verb/echo.py

```diff
@@ -76,6 +77,11 @@ def add_arguments(self, parser, cli_name):
             '--no-arr', action='store_true', help="Don't print array fields of messages")
         parser.add_argument(
             '--no-str', action='store_true', help="Don't print string fields of messages")
+        parser.add_argument( 
+            '--once', action='store_true', help="Print the first message received and then exit")
+        parser.add_argument(
+            '--timeout', metavar='N', type=unsigned_int, default=3.0,
```

Comment:
@craigh92 why 3 seconds? It seems a bit arbitrary to me. Consider making no timeout (indefinite spin) the default.

Reply:
I have now changed this so the default is no timeout, as suggested.